cg/meta/workflow/microsalt/microsalt.py

```diff
@@ -89,13 +89,16 @@ def get_config_path(self, filename: str) -> Path:
 
     def get_job_ids_path(self, case_id: str) -> Path:
         project_id: str = self.get_project_id(case_id)
-        return Path(
+        job_ids_path = Path(
             self.root_dir,
             "results",
             "reports",
             "trailblazer",
             f"{project_id}_slurm_ids{FileExtensions.YAML}",
         )
+        if job_ids_path.exists():
+            job_ids_path.unlink()
```

Comment:
I think this adds more functionality to the function than what can be understood from its name. 

Also, can we add a test case that catches the bug behavior?

Reply:
Also, slightly unclear why it is there without more context? Maybe we could break it out into a mini-method `_ensure_old_job_ids_file_removed` or something?

Reply:
Extracted a method with a more easily understandable name 👍  Won't add a test for this bug I don't given the circumstances and the type of bug this is. Won't change the name because the caller should not care and this methods is part of the workflow interface

Reply:
`get_job_ids_path` is a function commonly used in other `workflows`. The `get_job_ids_path` for microsalt now does not behave the same as the other 6 functions. I think this would warrant an added test so that in future if this code where to be changed the bug would be caught.